lms/static/scripts/frontend_apps/components/ErrorDisplay.js

```diff
@@ -1,5 +1,10 @@
 import { Link, Scrollbox } from '@hypothesis/frontend-shared';
 
+import { formatErrorDetails, formatErrorMessage } from '../errors';
```

Comment:
```suggestion
import { formatErrorDetails, formatErrorMessage } from '../errors';

```

Added a blank line after the imports.